lblod/frontend-embeddable-notule-editor
# Correct and document decision article uri generation

## Overview
Was using the default 'template style' URIs, this changes the default to a more usual format and documents how to change it. Also includes the latest plugins to get corrected snippet insertion URI instantiation.

##### connected issues and PRs:
Includes: https://github.com/lblod/ember-rdfa-editor-lblod-plugins/pull/498
Similar to: https://github.com/lblod/frontend-gelinkt-notuleren/pull/761
Part of https://binnenland.atlassian.net/browse/GN-5206

### Setup
N/A

### How to test/reproduce
Insert articles or snippets with articles and ensure that the URIs are generated correctly. Also check that passing a uriGenerator function to the besluit config works correctly.

### Challenges/uncertainties
N/A

### Checks PR readiness
